src/Azs.Compute.Admin/docs/Disable-AzsTenantSubscriptionFeature.md

```diff
@@ -0,0 +1,262 @@
+---
+external help file:
+Module Name: Azs.Compute.Admin
+online version: https://docs.microsoft.com/en-us/powershell/module/azs.compute.admin/disable-azstenantsubscriptionfeature
+schema: 2.0.0
+---
+
+# Disable-AzsTenantSubscriptionFeature
+
+## SYNOPSIS
+Disable the tenant subscription feature.
+
+## SYNTAX
+
+### DisableExpanded (Default)
+```
+Disable-AzsTenantSubscriptionFeature -FeatureName <String> [-Location <String>] [-SubscriptionId <String>]
```

Comment:
Can we rename this to Disable-AzsUserSubscriptionFeature ? User instead of Tenant